### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @javierskodnik

### CODE REVIEW

**Comment by javierskodnik** [comment]
@faloi ... me ovlidé de mencionarte en el momento que lo subí !!

File: README.md [hunk 1]
```diff
@@ -20,6 +20,5 @@ En ambos casos, incluir solamente los métodos y atributos que consideren import
 
 Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archivo desde un editor para notar la diferencia):
 
-![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
-![Desde un archivo local](./images/ejemplo-uml.png)
+![Desde un archivo local](./images/Obj2Tp1.png)
```
Comment (faloi) [inline README.md:24]: La información está bien pero el diagrama está raro, las flechas van al revés y ese `0..*` que pusiste tiene más pinta de DER que de diagrama de clases. Ya hablaremos sobre esto.

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,22 +1,32 @@
 class Barrio {
 
-	const property elementos = []
+	const property elementos = []   // lista
```
Comment (faloi) [inline src/barrio.wlk:3]: ¿Lista de qué?

File: src/barrio.wlk [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
 	}
 
+    //el método elementosBuenos() devuleve un númerico (la cantidad de elementos)
+
 	method elementosBuenos() {
 		return elementos.count{ e => e.esBueno() }
 	}
 
-	method elementosMalos() = elementos.count{ e => not e.esBueno() }
+    // el métod elementosMalos() no devuelve nada
```
Comment (faloi) [inline src/barrio.wlk:23]: Ups, sí devuelve, un número. No confundas el `=` con una asignación, en este caso es una sintaxis corta que tiene Wollok para definir un método que devuelve un valor.

File: src/elementos.wlk [hunk 4]
```diff
@@ -1,40 +1,47 @@
 class Hogar {
 
-	var property nivelMugre
-	var property confort
+	var property nivelMugre // variable numerica
+	var property confort // variable boolean
```
Comment (faloi) [inline src/elementos.wlk:4]: Nop, es numérica.

**Pull request closed without merging**